docs/cli-client/bank/account.md

```diff
@@ -0,0 +1,88 @@
+# iriscli bank account
+
+## Description
+
+Query  a special  account detial. 
+
+## Usage:
+
+```
+iriscli bank account [address] [flags] 
+```
+
+ 
+
+## Flags
+
+| Name,shorthand | Type   | Required | Default               | Description                                                  |
+| -------------- | ------ | -------- | --------------------- | ------------------------------------------------------------ |
+| -h, --help     |        | false    |                       | help for coin-type                                           |
```

Comment:
Help for account ? 
And let's update all `Description` begin with a capital letter, thanks.